### DISCUSSION TITLE
question about 24h2 atlas

### DISCUSSION THREAD

**Discussion by PouyaFakhari** [Atlas-OS/Atlas#1244] [Help & Support]
hello i use atlas os for a long time and its great!

but im on AtlasPlaybook_v0.3.2 for now and i recently update my windows 11 from 23h2 to 24h2
if  upcoming 24h2 playbook release can i use it?

im updating now and getting around 253 procces and i want to debloat my windows 24h2 

if you can pls make a playbook for people like me (im have a lot of apps and data and cant reinstall windows)
so if you can i will grateful if you make a play book for AtlasPlaybook_v0.3.2 for 24h2 so i can debloat my windows (or if i can use upcoming AtlasPlaybook let me know thanks)
![image](https://github.com/user-attachments/assets/57d4ad9b-f9ad-40fa-8f68-f14599c770a2)

**Comment by the-P1neapple** [comment]
Please see #1238
